plugins/destination/postgresql/client/migrate.go

```diff
@@ -107,25 +108,59 @@ func (c *Client) autoMigrateTable(ctx context.Context, table *schema.Table, chan
 	for _, change := range changes {
 		switch change.Type {
 		case schema.TableColumnChangeTypeAdd:
-			return c.addColumn(ctx, tableName, change.Current)
+			err := c.addColumn(ctx, tableName, change.Current)
+			if err != nil {
+				return err
+			}
+			continue
 		case schema.TableColumnChangeTypeRemove:
```

Comment:
Can we remove `case schema.TableColumnChangeTypeRemove` and `default:` altogether?